client.js

```diff
@@ -118,8 +119,23 @@ pc.script.create('client', function (context) {
 
             context.mouse.on('mousedown', this.onMouseDown, this);
             context.mouse.on('mouseup', this.onMouseUp, this);
-            
-            this.mouseState = false;
+
+            this.gamepadConnected = false;
+            this.gamepadActive = false;
+
+            window.addEventListener('gamepadconnected', function () {
```

Comment:
we probably should have all this code live in a separate file, but it's fine for now since the gamepad is changing [`movement[0]` and `movement[1]`](#diff-215b25d5bf5c0b4623ad1a2b62456f60R206), which get used later in the `update` method of `Client` to actually move the tank.